app/assets/stylesheets/master.css.scss

```diff
@@ -0,0 +1,21 @@
+.hero-image {
+  background: url('/assets/background-img.jpeg') no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  background-attachment: fixed;
+}
+
+.enter {
+  font-family: "Special Elite";
+  font-size: 45px;
+  color: #C9D1D4;
```

Comment:
Color literals like `#C9D1D4` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#C9D1D4` should be written as `#c9d1d4`